website_node_branded/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+====================
+website_node_branded
+====================
+
+Once installed, you can use the t-ignore-branding attribute on a <t> node to control if node is branded or qweb-active (bears a
+@data-oe-model or a @t-* *which is not t-field* as t-field does not section out views)
```

Comment:
@lmignon  Wouldn't it be nice to split   website_node_branded  in it's own PR?  I have some   documentation to contribute i think it would be cleaner./